templates/docs/examples/patterns/equal-height-row/4-items-per-column-wrap.html

```diff
@@ -0,0 +1,139 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Equal height row / Four items per column and full-screen divider, wrap{% endblock %}
+
+{% block standalone_css %}patterns_equal-height-row{% endblock %}
+
+{% block content %}
+<div class="p-equal-height-row--wrap has-divider-3">
+  <div class="p-equal-height-row__col">
+    <div class="p-equal-height-row__item">
+      <div class="p-image-container p-image-container--2-3 is-highlighted">
```

Comment:
`p-image-container--2-3` has all the styles from `p-image-container` because of the weird way we do BEM. So all that is needed is:

```suggestion
      <div class="p-image-container--2-3 is-highlighted">
```